rebasehelper/tests/test_specfile.py

```diff
@@ -341,6 +350,27 @@ def test_typo_fix_spec_hook(self, spec_object):
         assert '- This is changelog entry with some intentional typos' in spec_object.spec_content.sections[
             '%changelog']
 
+    def test_paths_to_rpm_macros_spec_hook(self, spec_object):
+        files = {
+            '/usr/share/man/man1/*': '%{_mandir}/man1/*',
+            '/usr/bin/%{name}': '%{_bindir}/%{name}',
+            '%config(noreplace) /etc/test/test.conf': '%config(noreplace) %{_sysconfdir}/test/test.conf',
+        }
+        files_devel = {
+            '/usr/share/test1.txt': '%{_datadir}/test1.txt',
+            '/no/macros/here': '/no/macros/here',
+        }
+        for path in files:
+            assert path in spec_object.spec_content.sections['%files']
+        for path in files_devel:
+            assert path in spec_object.spec_content.sections['%files devel']
+        PathsToRPMMacrosHook.run(spec_object, spec_object)
+
+        for expected_output in six.itervalues(files):
+            assert expected_output in spec_object.spec_content.sections['%files']
+        for expected_output in six.itervalues(files_devel):
+            assert expected_output in spec_object.spec_content.sections['%files devel']
+
```

Comment:
I think it would be better to compare the whole section, like in `test_spec_missing_file()` and `test_spec_missing_and_remove_file()`.

Reply:
Why do you think it is better? If you add a line to the section, you have to modify every test working with the section. This way it only checks the needed lines

Reply:
Because you are not really testing that the specified paths changed as expected (line index is discarded), if someone adds a different path that is equal to some of the current paths after processing, the test will no longer be reliable.
But I suppose it's not really such a big problem here.

Reply:
I didn't think about that, thanks for pointing it out. I will change the test to compare whole sections